components/recruitment/RoleCard.tsx

```diff
@@ -9,7 +9,7 @@ interface RoleCardProps {
 const RoleCard: React.FC<RoleCardProps> = ({ title, img }) => {
   return (
     <div className="relative z-[-10] m-16 flex h-2/3 w-2/3 items-center overflow-hidden rounded-lg bg-white p-4 shadow-xl">
-      <div>
+      <div className="max-w-2/3 w-2/3">
         <h1 className="mr-4 text-center text-xl font-semibold text-[#004C98]">
           {title}
```

Comment:
Could we make this text and background green for operations wing as shown in the Figma?
![image](https://github.com/NUS-Fintech-Society/SD_Fintech_Website_Revamp/assets/8915196/06fff8c9-b606-4dc5-98e4-1fdc8c1ee34c)

Reply:
Done!